lms/djangoapps/teams/models.py

```diff
@@ -88,6 +91,14 @@ class Meta(object):
     user = models.ForeignKey(User)
     team = models.ForeignKey(CourseTeam, related_name='membership')
     date_joined = models.DateTimeField(auto_now_add=True)
+    last_activity_at = models.DateTimeField()
+
+    def save(self, *args, **kwargs):
+        """ Customize save method to set the last_activity_at if it does not currently exist. """
+        if not self.last_activity_at:
```

Comment:
@dianakhuang explained to me-- if you use auto_now_add, the field is not editable. We do want to be able to change the last_activity_at date.